KsApi/GraphSchema.swift

```diff
@@ -3,7 +3,10 @@ import Prelude
 // MARK: - Graph Response
 
 public struct GraphResponse<T: Decodable>: Decodable {
-  let data: T?
+  let data: T
```

Comment:
Here we decouple `data` and `errors` by removing the `errors` from `GraphResponse` and giving them their own `GraphResponseErrors` struct. This allows us to decode errors separately from `data`.